packages/thrift-server-core/src/main/Int64.ts

```diff
@@ -247,6 +247,15 @@ export class Int64 implements IInt64 {
     // Min integer value that JS can accurately represent
     public static MIN_INT: number = -Math.pow(2, 53)
 
+    public static MAX_64_BIT_INTEGER = 9_223_372_036_854_775_807n
+    public static MIN_64_BIT_INTEGER = -9_223_372_036_854_775_808n
+
+    public static assert64BitRange(num: bigint): void {
```

Comment:
I started by adding this method in `utils` that created a circlular `requrie` in the generated JS files that because of the way TypeScript enums are generated resulted in null pointer exceptions.

Storing here seems okay for now. Will revisit if we ever remove `Int64` support.

Reply:
that is wild lol